src/v/transform/transform_processor.cc

```diff
@@ -228,8 +228,12 @@ ss::future<> processor::run_transform_loop() {
         co_await _engine->transform(
           std::move(*batch),
           _probe,
-          [&transformed](model::transformed_data data) {
+          [&transformed](
+            std::optional<model::topic_view> topic,
+            model::transformed_data data) {
+              vassert(topic == std::nullopt, "not supported yet 🙂");
```

Comment:
`"not supported yet 🙂");` ha, i can't tell if github is rendering this from ascii or its actually some unicode thingy.

Reply:
It's a real emoji :)